2024/political-donations-by-party/meta_data.json

```diff
@@ -0,0 +1,4 @@
+{
+    "description": "Donations to Republication candidates and PACs vs Democratic from 2015 to 2023",
```

Comment:
It looks like you're using the `to` instead of the hyphen here. That feels like the right move.